apps/condo/domains/ticket/schema/Ticket.js

```diff
@@ -424,6 +425,7 @@ const Ticket = new GQLListSchema('Ticket', {
                 overrideTicketFieldsForResidentUserType(context, resolvedData)
                 await setSectionAndFloorFieldsByDataFromPropertyMap(context, resolvedData)
                 setClientNamePhoneEmailFieldsByDataFromUser(get(context, ['req', 'user']), resolvedData)
+                await setDeadline(resolvedData)
```

Comment:
it's better to write something like:

if (!get(resolvedData, 'deadline') && !get(existingItem, 'deadline')) awawit setDeadline(...)

Probably, someone can use our API and can forgot to set this field